py3-cffi.yaml

```diff
@@ -0,0 +1,43 @@
+# Generated from https://pypi.org/project/cffi/
+package:
+  name: py3-cffi
+  version: 1.15.1
+  epoch: 0
+  description: Foreign Function Interface for Python calling C code.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - py3-pycparser
+      - python-3.11
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3.11
+      - python-3.11-dev
```

Comment:
do these need to be python 3.11?  Or can they track the latest python? i.e. dropping the `.11` similar to other examples
```
https://github.com/wolfi-dev/os/blob/d28eeb644d789dd48dfe39aa70fc29ba44672ddb/py3-pluggy.yaml#L19-L21
```

Reply:
let's give it a try, actually this content generated by the `melange convert python` then I didn't change any of the packages afterwards.